client/components/cocktail-list.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+class Cocktails extends React.Component {
+
+  constructor (props) {
+    super(props);
+    this.state = {
+      list: props.cocktails
+    };
+  }
+
+  _addCocktails () {
+    return this.props.cocktails.map(function (cocktail) {
```

Comment:
Oh yeah, just for consistency's sake, let's make this an arrow function: `this.state.cocktails.map((cocktail) => {...`